src/components/CardList/index.jsx

```diff
@@ -1,17 +1,16 @@
 import React, { useEffect, useState } from 'react';
 import Movie from '../Card';
-import fetchFrom from '../../services';
+import { fetchFrom } from '../../services';
 import './styles.css';
 
 function CardList({
   id,
   title,
   url,
   options,
+  setMovieInfo,
 }) {
   const [movieList, setMovieList] = useState([]);
-  const [maxPages, setMaxPages] = useState(0);
-  const [currentPage, setCurrentPage] = useState(options.page);
   const initialPosition = 40;
   const idName = `${id}-cardList`;
   const containerName = `${id}-containerCardList`;
```

Comment:
try to avoid declare so many variables in the component's body, this can cause bugs cos the overriding of the variables on rendering, use the useState and useEffect hooks to avoid that